tools/cleanup/cleanup.go

```diff
@@ -250,15 +250,15 @@ func (d *ImageDeleter) DeleteResources(project string, hoursToKeepResource int,
 			}
 		}
 		return nil
-	}, google.WithAuthFromKeychain(auther))
+	}, google.WithAuthFromKeychain(author))
```

Comment:
This is not author. :)

```suggestion
	}, google.WithAuthFromKeychain(auther))
```